Week3/More JavaScript.js

```diff
@@ -42,11 +42,28 @@ let vehicleCategory = {
     1 : "car",
     2 : "motorbike", 
 };
-    function vehicleType(color,category) {
-        console.log("a " + color + " " + vehicleCategory[category] );        
+    function vehicleType(color, category) {
```

Comment:
Ah, I think that you have too many `tabs` before your function. If you change it to this it will be easier to read:

```
function vehicleType(color, category) {
  if (category <= 2) {
    console.log('a ' + color + ' ' + vehicleCategory[category]);
  } else {
    console.log('Please enter valid');
  }
}
```

Reply:
Sorry, I just installed the Prettier extension :)